mars/dataframe/expressions/arithmetic/__init__.py

```diff
@@ -14,3 +14,15 @@
 
 from .abs import abs, DataFrameAbs
 from .add import add, DataFrameAdd
+
+
+def _install():
+    from ...core import DATAFRAME_TYPE
+    from .add import add, radd
+    for cls in DATAFRAME_TYPE:
+        setattr(cls, '__add__', add)
```

Comment:
`add` and `radd` method should also be added to `DATAFRAME_TYPE`.

Reply:
Like I said, `setattr(cls, 'add', add')` can be added as well, it's not necessary to define them on the classes.